apps/etterlatte-behandling/src/main/kotlin/grunnlagsendring/GrunnlagHelper.kt

```diff
@@ -81,4 +82,20 @@ fun Grunnlag.vergemaalellerfremtidsfullmakt(saksrolle: Saksrolle) =
         else -> throw IllegalArgumentException(
             "Prøvde å finne vergemål på en person som ikke er søker, men det er ikke relevant"
         )
+    }
+
+fun Grunnlag.sivilstand(saksrolle: Saksrolle) =
+    when (saksrolle) {
+        Saksrolle.SOEKER -> {
+            soeker.hentSivilstand()?.verdi
+        }
+        Saksrolle.GJENLEVENDE -> {
+            hentGjenlevende().hentSivilstand()?.verdi
+        }
+        Saksrolle.AVDOED -> {
+            hentAvdoed().hentSivilstand()?.verdi
+        }
+        else -> throw IllegalArgumentException(
+            "Prøvde å finne sivilstand for $saksrolle, men det er ikke relevant for denne rollen"
```

Comment:
Hvis det ikke er relevant burde man kanskje ikke kaste exception?

Reply:
tja.. fulgte bare samme konsept som det var gjort for de andre hendelsene. Dette gjelder for rollene SOESKEN og BARN så lite relevant og returnere noe for disse personene når det kommer til sivilstand.